doc/developer/projects.rst

```diff
@@ -119,17 +60,32 @@ Implement the VF2++ Graph Isomorphism Algorithm
 .. _`Graph Isomorphism Problem`: https://en.wikipedia.org/wiki/Graph_isomorphism_problem
 .. _VF2++: https://doi.org/10.1016/j.dam.2018.02.018
 
+Completed Projects
```

Comment:
@MridulS @dschult Would you mind adding the information about previous mentored projects.

Maybe
- https://www.cs.unm.edu/~bedwards/gsoc1.html
- https://www.google-melange.com/archive/gsoc/2015/orgs/python/projects/mridulseth.html

Also, is there more information we could link to?  It might be useful for future students to see proposals.  Are there any final reports?

I wonder if one or more of the current students would be willing to write a short "Advert"?  For example, they could discuss how it was working with us and any recommendations they would have to future students.  Ideally, the could write something that might increase the number of applicants as well as provide suggestions to the students so they make the most of the opportunity.

Reply:
> Also, is there more information we could link to? It might be useful for future students to see proposals. Are there any final reports?

I remember my proposal being in the Wiki section. Do you still have the dump from the GitHub wiki?